content/syllabuses/copywriting.md

```diff
@@ -1,6 +1,47 @@
----
-title: Copywriting
-_db_id: 5
----
+# **Welcome To The Wonderful World Of Copywriting!**
 
-Nothing to see here... yet
\ No newline at end of file
+### **Introduction and young explanation**
+
+## **So like... what is copywriting?**
+That is a good question! Now we are forced to give you a good answer -_-
+
+Before I waste too many words tryig to explain how we cast spells using the magic of words, here is a young video or two to help you understand...
+
+**Resources:**
+
+- [What is copywriting?](https://youtu.be/k5uI0yoKnF4)
+- [Practical example](https://youtu.be/MkF7X-tPLyU)
+
+Present Using This [Figma Template](https://www.figma.com/file/8JThmrfCGZGjajrxVB1YrX/Presentation-Template?node-id=0%3A1) You Can Only View So Duplicate It And Share Your Figma File Link
+
+## **The Don Draper Dream**
```

Comment:
There is no need to do headings like this.

```
## **The Don Draper Dream**
```
Those 2 hashes mean it's a aheading, i will be styled as a heading. The stars are just extra noise. If the styling is not cool whenb you leave out the stars let me know and it'll be a quick fir to css.

This is better:

```
## The Don Draper Dream
```